signup/tool/src/webapp/signup/newMeeting/step1.jsp

```diff
@@ -27,7 +27,7 @@
                     useTime: 1,
                     parseFormat: 'YYYY-MM-DD HH:mm:ss',
                     allowEmptyDate: false,
-                    val: '<h:outputText value="#{NewSignupMeetingBean.signupMeeting.startTime}"><f:convertDateTime pattern="yyyy-MM-dd HH:mm:ss"/></h:outputText>',
+                    val: '<h:outputText value="#{NewSignupMeetingBean.signupMeeting.startTime}"><f:convertDateTime pattern="yyyy-MM-dd HH:mm:ss" timeZone="#{NewSignupMeetingBean.signupMeeting.timezone}" /></h:outputText>',

```

Comment:
Can you just use UserTimeZone.userTimeZone like in tool/src/webapp/signup/signupMeetings.jsp ?